videos/management/commands/copy_videos.py

```diff
@@ -101,141 +86,7 @@ def handle(self, **options):
-            )
-            .execute()
-        )
-        return new_file.get("id")
-
-    def update_transcript_and_captions(
-        self, resource, new_transcript_file, new_captions_file
-    ):
-        """
-        Update the associated transcript and captions files for a resource.
-        """
-        resource.metadata["video_files"][
-            "video_transcript_file"
-        ] = f"/{str(new_transcript_file).lstrip('/')}"
-        resource.metadata["video_files"][
-            "video_captions_file"
-        ] = f"/{str(new_captions_file).lstrip('/')}"
-
-        resource.save()
-
-    def create_drivefile(self, gdrive_file, new_resource, destination_course):
-        """
-        Create a DriveFile for gdrive_file in the destination course.
-        """
-        gdrive_dl = get_gdrive_file(gdrive_file.get(DRIVE_FILE_ID))
-        DriveFile.objects.create(
-            website_content=new_resource,
-            file_id=gdrive_file.get(DRIVE_FILE_ID),
-            checksum=gdrive_dl.get(DRIVE_FILE_MD5_CHECKSUM),
-            name=get_resource_name(new_resource),
-            mime_type=new_resource.metadata["file_type"],
-            status=DriveFileStatus.COMPLETE,
-            website=destination_course,
-            s3_key=str(new_resource.file).lstrip("/"),
-            resource=new_resource,
-            drive_path=f"{destination_course.short_id}/{DRIVE_FOLDER_FILES_FINAL}",
-            modified_time=gdrive_dl.get(DRIVE_FILE_MODIFIED_TIME),
-            created_time=gdrive_dl.get(DRIVE_FILE_CREATED_TIME),
-            size=gdrive_dl.get(DRIVE_FILE_SIZE),
-            download_link=gdrive_dl.get(DRIVE_FILE_DOWNLOAD_LINK),
-            sync_dt=now_in_utc(),
-        )
-
-    def copy_video_resource(self, source_course, destination_course, source_resource):
-        """
-        Copy a video resource and associated captions/transcripts.
-        """
-
-        video_transcript_file = source_resource.metadata["video_files"][
-            "video_transcript_file"
-        ]
-        video_captions_file = source_resource.metadata["video_files"][
-            "video_captions_file"
-        ]
-        new_resource = create_new_content(source_resource, destination_course)
-        if video_transcript_file and video_captions_file:
-            video_transcript_resource = WebsiteContent.objects.filter(
-                file=video_transcript_file
-            ).first()
-            new_transcript_resource = create_new_content(
-                video_transcript_resource, destination_course
-            )
-            new_transcript_file = new_transcript_resource.file
-
-            video_captions_resource = WebsiteContent.objects.filter(
-                file=video_captions_file
-            ).first()
-            new_captions_resource = create_new_content(
-                video_captions_resource, destination_course
+            copy_video_resource.delay(
+                source_course.uuid, destination_course.uuid, video.text_id
```

Comment:
Maybe we can add some logging here, so it's more apparent that progress is being made? Currently running the command returns nothing. For example, `backpopulate_pipelines` does something like this:

```python
        self.stdout.write(
            f"Started celery task {task} to upsert pipelines for {len(website_names)} sites"  # noqa: E501
        )
```

It gives a little bit more assurance that the command is doing what it's supposed to do, and also gives you the UUID of the task in case you need to jump in a shell and troubleshoot. The `backpopulate_pipelines` command will also print errors from the task if there are any and print a message when the task is complete.